api/app/models/concerns/spree/webhooks/has_webhooks.rb

```diff
@@ -21,15 +21,23 @@ def event_name(operation)
         "#{self.class.name.demodulize.tableize.singularize}.#{operation}"
       end
 
-      def body
-        resource_serializer.new(self).serializable_hash.to_json
+      def body(event)
+        resource_serializer.new(self, serializer_params(event)).serializable_hash.to_json
       end
 
       def resource_serializer
         demodulized_class_name = self.class.to_s.demodulize
         "Spree::Api::V2::Platform::#{demodulized_class_name}Serializer".constantize
       end
 
+      def serializer_params(operation)
```

Comment:
```suggestion
      def serializer_params(operation:)
```
I can imagine that in future we would like to pass there the user / app who triggered the event, etc

Reply:
@damianlegawiec yes indeed.